<launch>
  <group ns="renov_up_level">
    <!-- mat document reading parameters -->
    <param name="mat_data_path" value="/home/zy/catkin_ws/src/paintingrobot/painting_robot_demo/data/data.mat"/>
    <param name="coverage_planner_path" value="/home/zy/catkin_ws/src/paintingrobot/painting_robot_demo/scripts"/>
    
    <!-- <param name="mat_data_path" value="/data/ros/renov_robot_ws/src/painting_robot_demo/data/data.mat"/> -->
    <!-- <param name="coverage_planner_path" value="/data/ros/renov_robot_ws/src/painting_robot_demo/scripts"/> -->
  
    <param name="mat_parameterx" value="0.430725381079" />
    <param name="mat_parametery" value="-0.00033063639818" />
    <param name="mat_parameterz" value="0.028625" />
    <param name="mat_interval" value="0.10" />



    <node pkg="painting_robot_demo" type="paintingrobotplanner_moveit_visualization.py" name="coverage_painting_simulation" output="screen" >

    <!-- <node pkg="painting_robot_demo" type="painting_endeffector_test.py" name="coverage_painting_simulation" output="screen" > -->
    </node>

  <!-- other parameters -->
  <!-- <param name="mobile_tracking_stop_flag" value="0" />
  <param name="painting_opreating_over" value="0"/> -->

  <!-- set /global_example parameter -->
  <!-- <include file="$(find painting_robot_demo)/launch/search_usb_port.launch">
  </include>
  <include file="$(find painting_robot_demo)/launch/plc_node_zy.launch">
  </include> -->
  <!-- <include file="$(find painting_robot_demo)/launch/mobile_platform_driver.launch">
  </include> -->
  <!-- <include file="$(find painting_robot_demo)/launch/3dof_mechanism_startup_zy.launch">
  </include>
  <include file="$(find painting_robot_demo)/launch/aubo_startup_zy.launch">
  </include>  -->

  </group>
</launch>

